src/ui-test/tests/remote.debug.on.openshift.test.ts

```diff
@@ -0,0 +1,245 @@
+
+
+describe.only('Remote debug on OpenShift', function () {
+    this.timeout(200000);
+
+    let driver: WebDriver;
+    let textEditor: TextEditor;
+
+    describe('Spring Boot project', function () {
+
+        before(async function () {
+            await prepareEnvironment('springboot', SPRINGBOOT_DIR);
+        });
+
+        after(async function () {
+            await cleanUp(SPRINGBOOT_DIR);
+        });
+
+        it(`SpringBoot`, async function () {
+            driver = VSBrowser.instance.driver;
+
+            await createCamelRoute(driver, "Demo");
+            await createProject(driver, SPRINGBOOT_CREATE_COMMAND)
+
+            // open file
+            await VSBrowser.instance.openResources(path.join(SPRINGBOOT_PROJECT_FOLDER, DEMO_FILE));
+
+            // set breakpoint
+            textEditor = new TextEditor();
+            await driver.wait(async function () {
+                return await textEditor.toggleBreakpoint(15);
+            }, 5000);
+
+            // open debug view
+            const debugView = await openDebugView();
+
+            // validate configuration and start debug
+            assert.isTrue(await validateLaunchConfiguration(debugView, SPRINGBOOT_ATTACH_DEBUGGER));
+            await startDebugSession(debugView,SPRINGBOOT_ATTACH_DEBUGGER);
+           
+            const terminalView = await new BottomBarPanel().openTerminalView();
+
+            // check breakpoint is hitted
+            const breakpoint = await waitForBreakpointPause(driver, textEditor);
+            assert.isTrue(await breakpoint.isPaused());
+        });
+    });
+
+    describe('Quarkus project', function () {
+
+        before(async function () {
+            await prepareEnvironment('quarkus', QUARKUS_DIR);
+        });
+
+        after(async function () {
+            await cleanUp(QUARKUS_DIR);
+        });
+
+        it(`Quarkus`, async function () {
+            driver = VSBrowser.instance.driver;
+
+            await createCamelRoute(driver, "Demo");
+            await createProject(driver, QUARKUS_CREATE_COMMAND);
+
+            // open file
+            await VSBrowser.instance.openResources(path.join(QUARKUS_PROJECT_FOLDER, DEMO_FILE));
+
+            // set breakpoint
+            textEditor = new TextEditor();
+            await driver.wait(async function () {
+                return await textEditor.toggleBreakpoint(12);
+            }, 5000);
+
+            // open debug view
+            const debugView = await openDebugView();
+
+            // validate configuration and start debug
+            assert.isTrue(await validateLaunchConfiguration(debugView, QUARKUS_ATTACH_DEBUGGER));
+            await startDebugSession(debugView,QUARKUS_ATTACH_DEBUGGER);
+
+            const terminalView = await new BottomBarPanel().openTerminalView();
+
+            // check breakpoint is hitted
+            const breakpoint = await waitForBreakpointPause(driver, textEditor);
+            assert.isTrue(await breakpoint.isPaused());
```

Comment:
I think it would be convenient to have a message for the assertion explaining what is checked